apps/api/src/sockets/gateway.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  OnGatewayConnection,
+  OnGatewayDisconnect,
+  OnGatewayInit,
+  WebSocketGateway,
+  WebSocketServer,
+} from '@nestjs/websockets'
+import { JwtService } from '@nestjs/jwt'
+import { PersonService } from '../person/person.service'
+import { Person } from '@prisma/client'
+import { Logger } from '@nestjs/common'
+
+@WebSocketGateway({ namespace: '/api/v1', transport: 'websocket' })
+export class SocketGateway implements OnGatewayConnection, OnGatewayInit, OnGatewayDisconnect {
```

Comment:
The general naming guideline is the name to show the intended usage of the class.
In this case:
The purpose of this gateway is to Notify, hence NotificationGateway is a better name. In contrast, a SocketGateway name shows only what it is made of, but not what it is used for.